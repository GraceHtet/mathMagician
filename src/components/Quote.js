import quoteStyle from './styles/quote.module.css';

const Quote = () => (
  <div className={quoteStyle.quote}>
    Mathematics is not about numbers, equations, computations, or alogrithms: It
    is about understanding -William Paul Thurston
  </div>
);

export default Quote;
