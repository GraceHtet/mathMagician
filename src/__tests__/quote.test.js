import * as React from 'react';
import { render, screen } from '@testing-library/react';
import Quote from '../components/Quote';

describe('Quote', () => {
  test('Quote Text match SnapShot', () => {
    render(<Quote />);
    expect(
      screen.getByText(
        /Mathematics is not about numbers, equations, computations, or alogrithms: It is about understanding -William Paul Thurston/i,
      ),
    ).toMatchSnapshot();
  });
});
