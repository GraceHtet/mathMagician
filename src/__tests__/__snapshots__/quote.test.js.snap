// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Quote Text match SnapShot 1`] = `
<div
  class="quote"
>
  Mathematics is not about numbers, equations, computations, or alogrithms: It is about understanding -William Paul Thurston
</div>
`;
